components/style/values/computed/pointing.rs

```diff
@@ -15,15 +15,17 @@ use std::fmt::{self, Write};
 use style_traits::{CssWriter, ToCss};
 use style_traits::ParseError;
 use style_traits::cursor::CursorKind;
+use values::computed::color::Color;
+use values::generics::pointing::CaretColor as GenericCaretColor;
+#[cfg(feature = "gecko")]
+use values::specified::url::SpecifiedUrl;
 
 /// The computed value for the `cursor` property.
 ///
 /// https://drafts.csswg.org/css-ui/#cursor
 pub use values::specified::pointing::Cursor;
 #[cfg(feature = "gecko")]
 pub use values::specified::pointing::CursorImage;
-#[cfg(feature = "gecko")]
-use values::specified::url::SpecifiedUrl;
```

Comment:
Hm? Was this unused? How does this pass CI?

Reply:
I moved it in previous block because there are 2 actual lists of uses, with the second one supposed to contain reexported stuff with their own doc block. TL;DR it's a mess.

Reply:
Oh I'm blind, nvm, thanks for moving it.